docs/content/tutorials/tutorial.md

```diff
@@ -22,6 +22,14 @@ services:
       - MYSQL_ROOT_PASSWORD=123456
       - MYSQL_USER=mysqluser
       - MYSQL_PASSWORD=mysqlpw
+  mongo:
+    image: "mongo:4.0-xenial"
+    command: --replSet rs0 --smallfiles --oplogSize 128
+    ports:
+      - "27017:27017"
+    environment:
+      - MONGO_INITDB_ROOT_USERNAME=mongouser
+      - MONGO_INITDB_ROOT_PASSWORD=mongopw
```

Comment:
I have updated the existing tutorial pages to have a better meaningful title. You can create a new page for mongodb, the title can be "Streaming ETL from MongoDB to Elasticsearch"

Reply:
Thanks @wuchong, I'll move the tutorial to that.